babel.config.js

```diff
@@ -1,96 +1,11 @@
-const browsers = require('./browserslist');
-const plugins = [
-    ['@babel/plugin-proposal-class-properties'],
-    ['@babel/plugin-proposal-object-rest-spread'],
-    ['@babel/plugin-syntax-dynamic-import'],
-    ['@babel/plugin-syntax-jsx'],
-    ['@babel/plugin-transform-react-jsx'],
-    ['babel-plugin-graphql-tag']
-];
-
-const targets = {
-    dev: 'last 2 Chrome versions',
-    prod: browsers,
-    test: 'node 10'
+module.exports = api => {
+    // const env = api.env() || 'development';
+    const config = { presets: ['@magento/peregrine'] };
+    if (process.env.CODEBUILD_BUILD_ID) {
```

Comment:
What does this signify?

Reply:
That env var is present in AWS CodeBuild. I'm trying to make sure we don't leave stale Babel cache in the build, which has happened before.